src/components/Charger.js

```diff
@@ -2,6 +2,10 @@ import { Card, CardContent, Typography, CardMedia } from "@mui/material";
 import { Link } from "react-router-dom";
 
 export const Charger = ({ charger }) => {
+
+    //TODO: handle price display
+  const formatedPrice = `$${charger.price/100}`;
```

Comment:
There's a little helper method in utils/helpers you can import to handle conversions for this 😄